docker-machine-nfs.sh

```diff
@@ -146,15 +146,15 @@ configureNFS()
     echoSuccess "\t\t\t\t\t\tOK"
 }
 
-# @info:    Configures the Boot2Docker to mount nfs
-configureBoot2Docker()
+# @info:    Configures the Docker Machine to mount nfs
+configureDockerMachine()
```

Comment:
Does it though? Semantically, as I understand it, it's a Docker Machine. The filesystem still references boot2docker (for now), but I thought it was named a Docker Machine.